x-pack/plugins/fleet/server/services/api_keys/security.ts

```diff
@@ -64,7 +64,7 @@ export async function authenticate(callCluster: CallESAsCurrentUser) {
   }
 }
 
-export async function invalidateAPIKey(soClient: SavedObjectsClientContract, id: string) {
+export async function invalidateAPIKey(soClient: SavedObjectsClientContract, ids: string[]) {
```

Comment:
Since this now accepts `ids: string[]` I think the function should be pluralized to `invalidateAPIKeys`